source/user-manual/user-administration/password-management.rst

```diff
@@ -213,12 +213,17 @@ Follow the instructions below to change the passwords for all the Wazuh indexer
       INFO: The password for Wazuh API user wazuh is ivLOfmj7.jL6*7Ev?UJoFjrkGy9t6Je.
       INFO: The password for Wazuh API user wazuh-wui is fL+f?sFRPEv5pYRE559rqy9b6G4Z5pVi
 
-#. On `all your Wazuh server nodes`, run the following command to update the `admin` password in the Filebeat keystore and in the ``ossec.conf`` file for the Wazuh server. Replace ``<ADMIN_PASSWORD>`` with the random password generated in the first step.
+#. On `all your Wazuh server nodes`, run the following command to update the `admin` password in the Filebeat keystore. Replace ``<ADMIN_PASSWORD>`` with the random password generated in the first step.
 
    .. code-block:: console
 
       # echo <ADMIN_PASSWORD> | filebeat keystore add password --stdin --force
```

Comment:
This step would not be necessary in this part. This command should be executed if the user configured a custom user for Filebeat

` # echo <CUSTOM_USER_PASSWORD> | filebeat keystore add password --stdin --force`